appinventor/components/src/com/google/appinventor/components/runtime/PointChartDataModel.java

```diff
@@ -111,4 +73,9 @@ protected YailList getDefaultValues(int size) {
 
     return YailList.makeList(defaultValues);
   }
+
+  @Override
+  protected boolean areEntriesEqual(Entry e1, Entry e2) {
```

Comment:
Test cases:
- [ ] x values equal, y values equal
- [ ] x values equal, y values not equal
- [ ] x values not equal, y values equal